### PR TITLE
index places

### PR DESCRIPTION
**Pull request by JauneJaune**
List of places no filter

### CODE REVIEW

File: app/models/place.rb [hunk 1]
```diff
@@ -9,6 +9,5 @@ class Place < ActiveRecord::Base
   validates :capacity, presence: true
   validates :rooms, presence: true
   validates :beds, presence: true
-  validates :rate, presence: true
```
Comment (floangles) [inline app/models/place.rb:12]: You need to add rate validaiton

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -8,7 +8,15 @@
   <div class="container">
       <div class="row">
         <div class="col-sm-offset-3 col-xs-12">
-    <!-- ajouter le form ici  -->
+          <%= form_tag(places_path, method: "get") do %>
+            <%= label_tag('enter a city plz') %>
+            <%= text_field_tag(:city) %>
+            <%= label_tag('enter a number of rooms plz') %>
+            <%= text_field_tag(:rooms) %>
+            <%= label_tag('enter a number of beds plz') %>
```
Comment (floangles) [inline app/views/home/index.html.erb:16]: numbe field

**Comment by floangles** [comment]
à refaire

File: app/views/home/index.html.erb (hunk 2 above)
Comment (JauneJaune) [inline app/views/home/index.html.erb:16]: ok

File: app/models/place.rb (hunk 1 above)
Comment (JauneJaune) [inline app/models/place.rb:12]: ok

**Pull request merged**